src/Goteo/Library/Forms/Model/ImpactDataForm.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with this source code.
+ */
+
+namespace Goteo\Library\Forms\Model;
+
+use Goteo\Library\Forms\AbstractFormProcessor;
+use Goteo\Library\Forms\FormModelException;
+use Goteo\Library\Text;
+use Goteo\Model\Image;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Validator\Constraints;
+
+use Goteo\Model\Filter\ImpactData;
+
+class ImpactDataForm extends AbstractFormProcessor {
+
+    public function createForm() {
+
+        $model = $this->getModel();
+        $builder = $this->getBuilder();
+        $options = $builder->getOptions();
+        $data = $options['data'];
+
+        $builder
+            ->add('title', 'text', array(
```

Comment:
@davidbeig would you mind using the new Symfony Form syntax instead?

So instead of 'text' it would be TextType::class (you'd need to import the class as well). 

That way, we would have less work to do when we merge the Symfony 3.4 branch into the live branch. Which could happen before or after this one.

The same for the other types in this form.